freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -70,11 +68,37 @@ def handle(self, event):
             self.log_info(msg)
             return []
 
-        # Get original nvrs of all builds in the advisory
-        original_nvrs = [build.original_nvr for build in db_event.builds]
+        # Get builds NVRs from the advisory attached to the message/event
+        affected_nvrs = set()
+        for product_info in event.advisory.builds.values():
+            for build in product_info['builds']:
+                affected_nvrs.add(build['nvr'])
+
+        # Search for the first event that triggered the chain of rebuilds
```

Comment:
I don't think we need to get the event first, why not just get the original nvr in `models.ArtifactBuild`?

Reply:
It's because our models.Event probably will not have any builds attached(since we didn't do any? I am not sure). So by calling `event.advisory.builds` I am calling errata api to get builds for the attached advisory.

Reply:
Which event do you mean? When an advisory filed by botas is shipped, it always contains some images, an advisory can not be moved to QE or shipped without any build (except Text only advisories, but we're dealing with container advisories). In this shipped advisory, the attached images are always rebuilt by Freshmaker, for example, it contains an image `foobar-1.2-1.1607508530`, this image was rebuilt by Freshmaker and we can find the original NVR by  `ArtifactBuild.query.filter_by(rebuilt_nvr="foobar-1.2-1.1607508530").one()`.